src/i18n.js

```diff
@@ -1,17 +1,20 @@
+import en from '../i18n/en.json'
+
 const messages = {
-	en: {
-		'wikipedia': 'Wikipedia'
-	},
-	fr: {
-		'wikipedia': 'Wikipédia'
+		en
 	},
-	es: {
-		'wikipedia': 'Wikipedia'
-	}
-}
 
-const msg = (lang, key) => {
-	return (messages[lang] || messages.en)[key] || key
-}
+	msg = ( lang, key ) => {
+		try {
+			if ( !messages[ lang ] ) {
```

Comment:
I would suggest reworking this logic to not retry to load a missing language all the time. We could put the catch around the `require` and maybe initialize the messages with an empty object when it fails.

Reply:
update, thanks it looks better now.